### PR TITLE
Base project

### PR DESCRIPTION
**Pull request by Phan-Duy**
create footer/slider

### CODE REVIEW

File: src/app/_components/cart-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  const cartItems = false;
+
+  return (
+    <div className="absolute p-4 rounded-md shadow-[0_3px_10px_rgb(0,0,0,0.2)] bg-white top-12 right-0 flex flex-col gap-6 z-20">
+      {!cartItems ? (
+        <div className="">Cart is Empty</div>
+      ) : (
+        <div className=""></div>
+      )}
+    </div>
```
Comment (ghost) [inline src/app/_components/cart-modal.tsx:13]:
```suggestion
      {cartItems ? (
      <div className=""></div>
        
      ) : (
        <div className="">Cart is Empty</div>
      )}
    </div>
```

File: src/app/_components/nav-icons.tsx [hunk 2]
```diff
@@ -1,14 +1,64 @@
...
+
+  const router = useRouter();
+
+  const isLoggedIn = false;
+
+  const handleProfile = () => {
+    if (!isLoggedIn) {
+      router.push("/login");
+    }
+    setIsProfileOpen((prev) => !prev);
```
Comment (ghost) [inline src/app/_components/nav-icons.tsx:19]:
```suggestion
    setIsProfileOpen(!isProfileOpen);
```

File: src/app/_components/product-list.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+const PRODUCT_PER_PAGE = 8;
+
+const ProductList = () => {
+    return (
+        <div className="">ProductList</div>
+    );
+};
+
+export default ProductList;
```
Comment (ghost) [inline src/app/_components/product-list.tsx:10]: endline

**Comment by Phan-Duy** [comment]
finish comment

**Pull request merged**